src/svcutilcore/src/Microsoft/Tools/ServiceModel/SvcUtil/InputModule.cs

```diff
@@ -166,7 +166,7 @@ private static bool TryFindFiles(string path, out FileInfo[] fileInfos)
                 // Figure out the directory part
                 string dirPath = Path.GetDirectoryName(path);
                 if (string.IsNullOrEmpty(dirPath))
-                    dirPath = @".\";
+                    dirPath = string.Concat(@".", Path.DirectorySeparatorChar);
                 DirectoryInfo dirInfo = new DirectoryInfo(dirPath);
```

Comment:
You don't need the literal @ anymore as there is no \ which would be treated as an escape character. Also `Path.DirectorySeparatorChar` is a static readonly char populated from PathInternal.DirectorySeparatorChar which is a const. If you use `"." + Path.DirectorySeparaterChar` instead of explicitly calling string.Concat, I think (only 98% sure) that the JIT will combine them into a literal constant. Either way, string.Concat is usually not needed as the compiler will emit that if necessary. The main exception being passing an array or IEnumerable of strings to be concatenated.